crux-llvm/exe/Main.hs

```diff
@@ -1,3 +1,23 @@
 module Main (main) where
 
-import CruxLLVMMain (main)
+import Control.Monad
+import System.Exit
+import System.Posix.Signals
+import System.Posix.Process
+
+import CruxLLVMMain (mainWithOutputConfig, defaultOutputConfig)
+
+-- Convert a SIGTERM signal into a SIGINT signal for the entire process group
```

Comment:
Question here: why do we need to convert the SIGTERM into a SIGINT? It seems like the goal is to just broadcast the signal we got to the rest of the process group (which we have to do manually because the GHC RTS/process API have special handling for SIGINT).  If we just rebroadcast the SIGTERM, wouldn't that accomplish basically the same goal?

Reply:
I'll try that and see what happens.  The reason I converted to SIGINT was because I wanted to activate GHC's special handling (and the delegate_ctlc behavior), but maybe it's not necessary?

Reply:
Turns out, rebroadcasting SIGTERM also does the trick, and I think it's a better approach.